CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ ENDIF()
 IF(WIN32)
     SET(Boost_USE_STATIC_LIBS ON)
 ENDIF()
-FIND_PACKAGE(Boost REQUIRED COMPONENTS program_options)
+FIND_PACKAGE(Boost REQUIRED COMPONENTS program_options system)
```

Comment:
It seems that Boost.Thread depends on Boost.System. Hence the explicit requirement for it.